packages/cli/src/commands/governance/propose.ts

```diff
@@ -25,6 +25,17 @@ export default class Propose extends BaseCommand {
       description: 'Amount of Celo to attach to proposal',
     }),
     from: CustomFlags.address({ required: true, description: "Proposer's address" }),
+    useMultiSig: Flags.boolean({
+      description: 'True means the request will be sent through multisig.',
+    }),
+    for: CustomFlags.address({
+      dependsOn: ['useMultiSig'],
+      description: 'Address of the multi-sig contract',
+      /* 
+      TODO(Arthur): check this syntax is correct.
```

Comment:
please let us know if this is still a todo

Reply:
Great catch, removed in https://github.com/celo-org/developer-tooling/pull/233/commits/c85c7887e95ec5ddfcb53413a1dddd30d5746a15